lib/request.js

```diff
@@ -206,7 +206,7 @@ Object.defineProperties(Request.prototype, {
           get: () => context.config
         },
         schema: {
-          get: () => context.schema
+          get: () => context.schema ?? {}
```

Comment:
Isn't [this](https://github.com/fastify/fastify/pull/5395/files#diff-2871801a54601791d9e1004f04b2df5d7adc881560285ff3fa6bee7bf9697857R231) enough to not get undefined?

Looks like `?? {}` here is a duplicate

```suggestion
          get: () => context.schema
```